classes/01class/exercises/c01-e03/lgothelipe/c01-e03.txt

```diff
@@ -0,0 +1,10 @@
+Logged on my AWS account and launched a t2.micro instance under EC2 -> Instances.
+Selected default values and downloaded a key pair at the end.
+Converted the key .pem to .ppk using PuttyGen.
+Connected to the instance's public DNS address using Putty.
+
+Created a new EC2 t2.micro with Auto-assign Public IP option disabled using the same key as before.
+Uploaded the key file .pem using WinSCP to the 2nd EC2.
```

Comment:
Can you confirm to which instance you uploaded the pem file? if it was to the first(with public IP) or the second instance(no public IP)?

Reply:
I've uploaded the .pem file to the 1st EC2, the one with Public IP address using WinSCP so I could use the file to connect to the 2nd one (private IP address only)